### PR TITLE
Add Minecraft.app Dependency

### PR DESCRIPTION
**Pull request by SmolSoftBoi**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

**Comment by reitermarkus** [comment]
Doesn't it come with a vendored Java-Runtime?

**Comment by maxnordlund** [comment]
No it doesn't, unless that changed recently. But I'm not sure if the launcher needs a JVM or not. But the main game definitively does.

**Comment by reitermarkus** [comment]
`/Applications/Minecraft.app/Contents/runtime/jre-x64/1.8.0_74/bin/java`

**Comment by maxnordlund** [comment]
Wow, that's interesting for sure. But I guess it makes sense. Either way, this means it doesn't require Java, right?

**Comment by vitorgalvao** [comment]
> Either way, this means it doesn't require Java, right?

Looks like it.

File: Casks/minecraft.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@
   name 'Minecraft'
   homepage 'https://minecraft.net/'
 
+  depends_on cask: 'java'
```
Comment (vitorgalvao) [inline Casks/minecraft.rb]: Please don’t do this. Java is a too complicated beast.

The proper way to indicate a java dependency is [with a `caveat`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/caveats.md#caveats-mini-dsl).

**Pull request closed without merging**